src/lib/eina/eina_inline_lock_win32.x

```diff
@@ -26,6 +26,10 @@
 #include <synchapi.h>
 #undef WIN32_LEAN_AND_MEAN
 
+#ifndef _MSC_VER
```

Comment:
Same as in this [one](https://github.com/expertisesolutions/efl/pull/159/files#r449040104)